src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -46,13 +51,17 @@ public class Drivetrain extends SubsystemBase {
   private SlewRateLimiter m_rotLimiter;
 
   private Timer m_timer;
-  private double m_prevTime;
+  private double m_prevSlewRateTime;
```

Comment:
What is slew rate?

Reply:
change in voltage over a time step

Reply:
a limiter is important to prevent brownouts from spikes in voltage

Reply:
in this case the name change is to make clear that the previous time is only updated when slew rate limiting is used, and so only when the drivetrain is being given a movement command